src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt

```diff
@@ -142,6 +149,28 @@ class OrderlyServer(
         }
     }
 
+    override fun post(url: String, data: String, params: Map<String, String?>): OrderlyServerResponse
```

Comment:
For interface clarity, might be inclined to rename the `data` parameter to `json` since only json content type is supported

Reply:
I've given the `data` and `params` arguments more descriptive names